UI/app/src/main/java/com/example/pathfinder/Activities/DriverDashboard.java

```diff
@@ -124,28 +166,32 @@ public void onFailure(IMqttToken asyncActionToken, Throwable exception) {
         });
     }
 
-    /**Driving using thumbstick
+    /**
+     * Driving using thumbstick
      * If cruise control is enabled, speed will be set from seekBar's progress
-     * If cruise control is disabled, speed will be set as percentage of seekBar's progress*/
+     * If cruise control is disabled, speed will be set as percentage of seekBar's progress
+     */
     @Override
     public void onThumbstickMoved(float xPercent, float yPercent, int id) {
-        int angle = (int)((xPercent) * 100);
+        int angle = (int) ((xPercent) * 100);
         int strength;
         //We need the negative of seekBar.getProgress()
-        int seekProgress = - seekBar.getProgress();
-        if(isCruiseControl){
+        int seekProgress = -seekBar.getProgress();
```

Comment:
Here line 179 is a conflict with master because of (negative as it is) seekbar instance which is named in master before additional "m" in the front. Easy fix ;)